<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCorrectorDetails</class>
 <widget class="QMainWindow" name="FormCorrectorDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>660</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
 	background-color: rgb(255, 255, 255);
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPlotter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Plotter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTuning">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>82</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Tuning</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReadings">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>82</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Readings</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
    </property>
    <property name="title">
     <string>Details</string>
    </property>
    <widget class="QSimpleShape" name="ledON">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledOFF">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledFault">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QELabel" name="lblState">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getStateHandler</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Interlock</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnOFF">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>26</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnON">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>26</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltage</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblVoltage">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getVmon</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblBusVoltage">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getVbus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Bus Voltage</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>130</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reference Current</string>
     </property>
    </widget>
    <widget class="QENumericEdit" name="txtCurrent">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>60</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setReference</string>
     </property>
     <property name="leadingZeros">
      <number>2</number>
     </property>
     <property name="precision">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-99.998999999999995</double>
     </property>
     <property name="maximum">
      <double>99.998999999999995</double>
     </property>
     <property name="writeOnChange">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QELabel" name="lblCurrent">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getReference</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>260</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
    </property>
    <property name="title">
     <string>Readings</string>
    </property>
    <widget class="QELabel" name="lblTemperature">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getTemperature</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
    <widget class="QESimpleShape" name="ledIn1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>102</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getSlowInputs.B1</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-PS-HC1</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblIn1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getSlowInputs.B1ST</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-PS-HC1</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblIn2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getSlowInputs.B2ST</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-PS-HC1</string>
     </property>
    </widget>
    <widget class="QESimpleShape" name="ledIn2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>122</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getSlowInputs.B2</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-PS-HC1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Slow Inputs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Slow Outputs</string>
     </property>
    </widget>
    <widget class="QESimpleShape" name="ledOut2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>122</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getSlowOutputs.B2</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-PS-HC1</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblOut2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getSlowOutputs.B2ST</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-PS-HC1</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblOut1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getSlowOutputs.B1ST</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-PS-HC1</string>
     </property>
    </widget>
    <widget class="QESimpleShape" name="ledOut1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>102</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getSlowOutputs.B1</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-PS-HC1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>381</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
    </property>
    <property name="title">
     <string>Faults</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Fault</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFaults_0">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Faults History</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFaults_1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFaults_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFaults_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
